app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SessionsController < ApplicationController
+  def omniauth
+    user = User.from_omniauth(request.env['omniauth.auth'])
+    if user.valid?
+      session[:user_id] = user.id
```

Comment:
This looks good.  I'll work on getting the `Sessions` together so that we can create a new session when the user is valid.